### DISCUSSION TITLE
六分街日报    警惕深蓝骗局

### DISCUSSION THREAD

**Discussion by yutong903** [share121/inter-knot#3025] [其他]
近日，一起诈骗案引起了人们的广泛关注。
据受害者描述，此诈骗犯有一头显眼的蓝色长发，性别为女，身高1.80左右，自称波桑小姐，受害者说，波桑小姐是由他们的好朋友，桑博先生引荐的，可惜，这位桑博先生为了救他们已经不幸葬身在了空洞中。
![750198c1885c12e972fad1149a19aa6d_t0379598407821ba175](https://github.com/user-attachments/assets/2e38e0c7-ffd5-4fd4-8d0f-30a19ddc9212)
据描述，这位波桑小姐经桑博先生介绍和他们合作，他们跟随波桑小姐进入了空洞中，波桑小姐说他们负责搬运货物，在路上，几只上级以骸突然出现，他们刚要回头寻找波桑小姐，却发现她早就不见踪影。
就在这千钧一发之际，桑博先生突然出现，拯救了他们所有人，可惜，他却永远的留在了空洞中。
这起事件告诉我们，不要相信那些一本万利的事情，警惕骗局，要脚踏实地的前行，才能有收获。
那么今天的报道就到这里，我们明天再见<img src="https://bbs-static.miyoushe.com/static/2023/10/07/517616fc27ab885b28447813af1a840f_323677886540923380.png" alt="桑博谢谢老板" width="75" >

**Comment by shan-dgha** [comment]
小灰毛好玩吗<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火礼物" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火礼物" width="75" >

**Comment by guozichang2007** [comment]
其实是花火<img src="https://bbs-static.miyoushe.com/static/2024/01/31/42b5ef8bab5a58fbdd74f17c21c12420_2665774360840406961.png" alt="花火 略略略" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" >
